### PR TITLE
Fixed /n

### PR DESCRIPTION
**Pull request by pharnb**
@rtrp/admins

### CODE REVIEW

File: users/pharnb/index.md [hunk 1]
```diff
@@ -10,8 +10,7 @@ Ben Pharn
 
 I am a first year Master's student here at Oregon State University, having completed my undergraduate degree here as well.  I was born and raised in Southern California, where the majority of my family still resides.
 
-[Email]<a href="mailto:pharnb@oregonstate.edu" target="top"> pharnb@oregonstate.edu </a> (pharnb@oregonstate.edu)
-/n [Resume](./files/resume.pdf)
+[Email]<a href="mailto:pharnb@oregonstate.edu" target="top"> pharnb@oregonstate.edu </a> (pharnb@oregonstate.edu)  [Resume](./files/resume.pdf)
```
Comment (dougwoods3) [inline users/pharnb/index.md]: [Resume] is still on the same line as your email address. @pharnb go to your repository, navigate to your index.md file and it should show you a preview of what your page will look like (without placeholders for the images). From there, click on the pencil on the top right to make an edit.

Now, there are two "tabs" available. One of them should (effectively) say "edit" and the other "preview". So you can make an edit and preview it without committing anything.

Come by my office if this doesn't make sense.

Comment (dougwoods3) [reply users/pharnb/index.md]: @pharnb This review is still open.

**Comment by pharnb** [comment]
Still working on this hold on

**Comment by dougwoods3** [comment]
No worries. I was just noticing there are a few open pull requests and wanted to remind you that you had one.

**Comment by pharnb** [comment]
@rtrp/admins

**Pull request merged**